DesktopModules/Vanjaro/UXManager/Library/Resources/Scripts/Blocks/button.js

```diff
@@ -135,16 +135,11 @@
 						cssproperties: [
 							{ name: "font-size" }
 						],
-						unitOptions: true,
-						units: [
-							{ name: 'px' },
-							{ name: '%' },
-							{ name: 'em' }
-						],
+                        units: [
+                            { name: 'px', min: 10, max: 100, step: 1, value: 16 },
+                            { name: 'vw', min: 0.5, max: 10, step: 0.1, value: 1 },
```

Comment:
vw should be default or the 1st selected option. Please confirm.

Reply:
default changed to vw